docs/content/users/quickstart.md

```diff
@@ -375,6 +375,32 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
     For more advanced tasks like adding elasticsearch, building and watching storefront and administration, see [susi.dev](https://susi.dev/ddev-shopware-6).
 
+=== "Silverstripe"
+
+    ## Silverstripe
+
+    === "Silverstripe"
+
+        Use a new or existing Composer project, or clone a Git repository.
+
+        === "Composer"
+            ```bash
+            mkdir my-silverstripe-app
+            cd my-silverstripe-app
+            ddev config --project-type=silverstripe --docroot=public --create-docroot
+            ddev composer create --prefer-dist --no-install --no-scripts silverstripe/installer -y
+            ddev composer install
```

Comment:
```suggestion
            ddev composer create --prefer-dist --no-scripts silverstripe/installer -y
```
This workaround is not needed anymore

Reply:
Oh? Nice!
It always kinda bugged me that it was this two-step method. I took it pretty much one-on-one from the Laravel docs, so maybe all the quickstarts need to be looked at and simplified?